txbridge/src/main/java/org/jboss/jbossts/txbridge/outbound/BridgeXAResource.java

```diff
@@ -87,10 +88,9 @@ public BridgeXAResource(Uid externalTxId, BridgeWrapper bridgeWrapper)
      */
     private void writeObject(ObjectOutputStream out) throws IOException
     {
-        txbridgeLogger.logger.trace("BridgeXAResource.writeObject() for tx id="+externalTxId);
```

Comment:
Have a trace message for the transaction is still useful.

As an aside (I am not asking you to make a change, it is just a comment) these log messages are not conditional on whether trace is enabled.